src/calculateRentalCost.js

```diff
@@ -28,7 +28,28 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const value = 40;
+  let sum = 0;
```

Comment:
The variable 'sum' is unnecessary. You can return the calculated value directly in each condition.